File: problem_0001/elixir/main.exs
n = 1000
IO.puts Enum.filter(1..(n - 1), fn(m) ->
  (rem(m, 3) == 0) || (rem(m, 5) == 0)
end) |> Enum.reduce(fn(m, sum) ->
  m + sum
end)

File: problem_0001/go/main.go
package main

import (
  "fmt"
)

func main() {
  var sum int
  n := 1000
  for i := 0; i < n; i++ {
    if i%3 == 0 || i%5 == 0 {
      sum += i
    }
  }
  fmt.Print(sum)
}
